bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-dropdown/component.jsx

```diff
@@ -0,0 +1,519 @@
+    if (!this.state.isActionsOpen && this.state.showNestedOptions) {
+      return this.resetMenuState();
+    }
+
+    this.checkDropdownDirection();
+  }
+
+  resetMenuState() {
+    return this.setState({
+      isActionsOpen: false,
+      dropdownOffset: 0,
+      dropdownDirection: 'top',
+      dropdownVisible: false,
+      showNestedOptions: false,
+    });
+  }
+
+  getUsersActions() {
+    const {
+      intl,
+      currentUser,
+      user,
+      router,
+      isBreakoutRoom,
+      getAvailableActions,
+      handleEmojiChange,
+      getEmojiList,
+      getEmoji,
+      setEmojiStatus,
+      assignPresenter,
+      removeUser,
+      toggleVoice,
+      changeRole,
+    } = this.props;
+
+    const userActions =
+    {
+      openChat: {
+        label: () => intl.formatMessage(messages.ChatLabel),
+        handler: (router, user) => this.onActionsHide(router.push(`/users/chat/${user.id}`)),
+        icon: 'chat',
+      },
+      clearStatus: {
+        label: () => intl.formatMessage(messages.ClearStatusLabel),
+        handler: user => this.onActionsHide(setEmojiStatus(user.id, 'none')),
+        icon: 'clear_status',
+      },
+      setPresenter: {
+        label: () => intl.formatMessage(messages.MakePresenterLabel),
+        handler: user => this.onActionsHide(assignPresenter(user.id)),
+        icon: 'presentation',
+      },
+      remove: {
+        label: user => intl.formatMessage(messages.RemoveUserLabel, { 0: user.name }),
+        handler: user => this.onActionsHide(removeUser(user.id)),
+        icon: 'circle_close',
+      },
+      mute: {
+        label: () => intl.formatMessage(messages.MuteUserAudioLabel),
+        handler: user => this.onActionsHide(toggleVoice(user.id)),
+        icon: 'mute',
+      },
+      unmute: {
+        label: () => intl.formatMessage(messages.UnmuteUserAudioLabel),
+        handler: user => this.onActionsHide(toggleVoice(user.id)),
+        icon: 'unmute',
+      },
+      promote: {
+        label: () => intl.formatMessage(messages.PromoteUserLabel),
+        handler: user => this.onActionsHide(changeRole(user.id, 'MODERATOR')),
+        icon: 'promote',
+      },
+      demote: {
+        label: () => intl.formatMessage(messages.DemoteUserLabel),
+        handler: user => this.onActionsHide(changeRole(user.id, 'VIEWER')),
+        icon: 'user',
+      },
+    };
+
+    const actions = getAvailableActions(currentUser, user, router, isBreakoutRoom);
+
+    const {
+      allowedToChatPrivately,
+      allowedToMuteAudio,
+      allowedToUnmuteAudio,
+      allowedToResetStatus,
+      allowedToRemove,
+      allowedToSetPresenter,
+      allowedToPromote,
+      allowedToDemote,
+      allowedToChangeStatus,
+    } = actions;
+
+    const emojiMenuControls = {
+      setstatus: {
+        icon: 'user',
+        label: () => intl.formatMessage(messages.statusTriggerLabel),
+        handler: () => { this.setState({ showNestedOptions: true }); },
+        desc: 'set_status',
```

Comment:
You should add a `iconRight (string)` to the `DropdownListItem` or the `UserAction` that would add an icon to the right side.

To me adding a `desc` , which people would associate to a `description` to compare and add the arrow icon seems not intuitive nor extensible.